.github/workflows/test.yml

```diff
@@ -25,6 +25,19 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ./.github/actions
       - run: ./test/rpc_test.py
+  rpc-k8s:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: extractions/setup-just@v1
+      - uses: medyagh/setup-minikube@master
+      - run: pip install --upgrade pip
+      - run: pip install -e .
```

Comment:
I don't think this will do anything, as `just kd` is what mounts the (code) directories, so there will be nothing to install?